ee/ucp/kubernetes/storage/use-azure-disk.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Configuring Azure Disk Storage for Kubernetes
+description: Learn how to add persistent storage to your Docker Enterprise clusters running on Azure with Azure Disk. 
+keywords: Universal Control Plane, UCP, Docker EE, Kubernetes, storage, volume
+redirect_from:
+---
+
+Operators can provide persistent storage for workloads running on Docker
```

Comment:
Pedantic but with `Operators` growing in popularity as a [Kubernetes component](https://coreos.com/operators/) should we use `Platform operators` instead? Anticipating confusion in the future on "so is Azure storage like an operator we install?"

Reply:
Agree with ^ . Platform or Cluster operators would help clarify.